### DISCUSSION TITLE
Wrong URL / 404 not found https://oss.sonatype.org/service/local//staging/deployByRepositoryId

### DISCUSSION THREAD

**Discussion by MartinWitt** [jreleaser/jreleaser#1633] [Q&A] [answered]
Hey,

with our release script, we get a 404 not found error and somehow JReleaser builds this strange url with 2x `//`. 
Workflow run: https://github.com/INRIA/spoon/actions/runs/8570265903/job/23487810824

Log: [jreleaser-release (9).zip](https://github.com/jreleaser/jreleaser/files/14885453/jreleaser-release.9.zip)
Someone else also has this issue?

**Comment by aalmiray** [comment, marked as answer]
I see in the log that v1.7.0 was used. We fixed an issue with double / after that release. Please upgrade to latest (1.11.0) and try again.

**Comment by MartinWitt** [comment]
It worked thank you very much. 👍
